tests/test_metatile.py

```diff
@@ -41,32 +41,34 @@ def test_make_metatiles_multiple(self):
             self.assertEqual(json, z.open('0/0/0.topojson').read())
 
     def test_make_metatiles_multiple_coordinates(self):
-        # we need to be able to handle this so that we can do "cut out"
-        # overzoomed tiles at z>16.
+        # checks that we can make a single metatile which contains multiple
+        # coordinates. this is used for "512px" tiles as well as cutting out
+        # z17+ tiles and storing them in the z16 (z15 if "512px") metatile.
 
         json = "{\"json\":true}"
         tiles = [
-            dict(tile=json, coord=Coordinate(17, 123, 456),
+            # NOTE: coordinates are (y, x, z), possibly the most confusing
+            # possible permutation.
+            dict(tile=json, coord=Coordinate(123, 456, 17),
                  format=json_format, layer='all'),
-            dict(tile=json, coord=Coordinate(17, 123, 457),
+            dict(tile=json, coord=Coordinate(123, 457, 17),
```

Comment:
Good point! I shall try to remember that next time. It will save me a lot of debugging!